styles.css

```diff
@@ -372,6 +383,7 @@ input:focus {
 
 @media (min-width: 375px) {
   .heroMainContainer {
+    /* height: 70%; */
```

Comment:
Please do not leave any uncommented code in your codebase. In this way, you can keep your codebase clean, neat, organized and professional. If you want to access a code you used before, you can navigate through your git history with ```git commit <commitnumber>```